docs/decisions/04-web-finger.md

```diff
@@ -0,0 +1,73 @@
+# WebFinger
+
+27-Aug-2023
+
+> Translate `user@domain` mentions to actor profile URIs. -- [doc](https://docs.joinmastodon.org/spec/webfinger/)
+
+[Issue 79](https://github.com/Letterbook/Letterbook/issues/79)
```

Comment:
I feel like it would make more sense to break this ADR out into its own discussion (and thus it's own PR). It seems like what you're getting at is a plan to establish isolated API testing, as distinct from both unit tests and larger integration tests. I can get behind that, but it's really tied up with the specifics of webfinger in this case, and I think that detracts from the proposal.

Reply:
The parts that are specific to what you've done to test the webfinger endpoint can just go in a readme in the test project.

Reply:
Latest has the test data change 👍

Reply:
Latest has the docs moved 👍